models/stage.model.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const StageSchema = new Schema(
+  {
+    submissionType: {
+      type: [Schema.Types.ObjectId],
+      ref: "Submission",
+    },
+    status: Boolean,
+    reviewer: {
+      type: [Schema.Types.ObjectId],
+      ref: "User",
+    },
+    marksheet: {
+      type: [Schema.Types.ObjectId],
+      ref: "Marksheet",
+    },
+    result: String,
+    document: String,
+    createdAt: Date,
```

Comment:
Remove this createdAt attribute and pass 'timestamps: true' as an option to the schema. Then add this line before exporting the Stage model 'StageSchema.index({ createdAt: 1 });'. Then createdAt time will be auto-generated.